src/backend/db/lib/seed.ts

```diff
@@ -0,0 +1,20 @@
+import { KyselyDatabaseInstance } from "@/backend/lib";
+
+export type SeedResult = Awaited<ReturnType<typeof seed>>;
```

Comment:
What is the purpose of this file?

Reply:
currently it's just for testing, but would be easy to set up as a dev script for some test data

can move it to the test folder for now

Reply:
d85cfc0